### PR TITLE
Hw work with file

### PR DESCRIPTION
**Pull request by Likh-Alex**

### CODE REVIEW

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
please fix travis
also, let's look at the checklist points about the constant fields

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+        File fileFrom = new File(fromFileName);
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileFrom))) {
+            String line = reader.readLine();
+            int supply = 0;
+            int buy = 0;
+            while (line != null) {
+                String[] lineData = line.split(DATA_SEPARATOR);
+                if (lineData[0].equals(SUPPLY_KEY_WORD)) {
+                    supply += Integer.parseInt(lineData[1]);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/WorkWithFile.java:25]: let's make `0` and `1` constants as well

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+                } else if (lineData[0].equals(BUY_KEY_WORD)) {
+                    buy += Integer.parseInt(lineData[1]);
+                }
+                line = reader.readLine();
+            }
+            writeReportToFile(supply,buy,toFileName);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/WorkWithFile.java:31]:
```suggestion
            writeReportToFile(supply, buy, toFileName);
```

File: src/main/java/core/basesyntax/WorkWithFile.java [hunk 3]
```diff
@@ -1,8 +1,55 @@
...
+        } catch (IOException e) {
+            throw new RuntimeException("Can not read data from file: " + fileFrom, e);
+        }
+    }
 
+    public void writeReportToFile(int supply, int buy, String toFileName) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/WorkWithFile.java:37]: helper methods should be private

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/WorkWithFile.java (hunk 1 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/WorkWithFile.java]: done

File: src/main/java/core/basesyntax/WorkWithFile.java (hunk 3 above)
Comment (Likh-Alex) [reply src/main/java/core/basesyntax/WorkWithFile.java]: done as well

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**